exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/dml/ReturningTests.kt

```diff
@@ -157,4 +160,24 @@ class ReturningTests : DatabaseTestsBase() {
             assertTrue { result3.all { it == 0.0 } }
         }
     }
+
+    @Test
+    fun testCustomEntityIdColumnAccess() {
+        val tester = object : IdTable<String>() {
+
+            val value = varchar("value", 128)
+
+            override val primaryKey: PrimaryKey = PrimaryKey(value)
+            override val id: Column<EntityID<String>> = value.entityId()
+        }
+
+        withTables(tester) {
+            tester.insert {
+                it[tester.value] = "test-value"
+            }
+            val entry = tester.selectAll().first()
+            TestCase.assertEquals("test-value", entry[tester.value])
+            TestCase.assertEquals("test-value", entry[tester.id].value)
```

Comment:
Please move this to a more appropriate test file. Anything in `shared/entities` I guess.

Also, is there a reason `junit.framework.TestCase.assertEquals` is being used instead of regular `assertEquals()`?

Reply:
Haven't found a better place than `EntityBugsRegressionTest`...